test/integration/viewer_spec.mjs

```diff
@@ -454,4 +454,346 @@ describe("PDF viewer", () => {
       );
     });
   });
+
+  describe("Detail view on zoom", () => {
+    function setupPages(zoom, maxCanvasPixels, width, height) {
+      const { resolve, reject, promise } = Promise.withResolvers();
+
+      let pages;
+
+      beforeAll(async () => {
+        const pagesP = loadAndWait(
```

Comment:
Nit: Personally I'm finding these sort of one-letter abbreviations unclear when reading code, can we please just spell it out instead?
```suggestion
        const pagesPromise = loadAndWait(
```